Security/Fundamentals/Security_Principles.mediawiki

```diff
@@ -43,23 +43,25 @@ Changes are detailed in the [https://github.com/mozilla/wikimo_opsec/commits/mas
 '''Do not'''
 * Allow root access (e.g. via 'sudo') to all systems for all operation engineers, regardless of the actual services they maintain.
 * Give access "just in case".
-* Never revisit access rules for users.
+* Keep adding access rights to users without revisiting if the users still use previously granted access rights.
 
 = Defense in Depth =
 == Do not allow lateral movement ==
 ''Make it difficult or impossible for an attacker to move from one host in the network to another host.''
 
 '''Do'''
 * Use host-based firewall rules that self-protect the system regardless of the rules from the network equipment, or the location of the system.
-* Avoid usage of firewall rules that allow any inbound connection to connect to a listening port (such as OpenSSH, RDP).
+* Avoid usage of firewall rules that allow any inbound connection to connect to a listening port.
```

Comment:
Sounds like I can't run a service on 443. Maybe "allow any inbound connection to a non-client-service port such as OpenSSH or RDP"